pkgs/applications/editors/pixelorama/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib , stdenv , fetchurl , godot-headless , godot-export-templates , writeShellScript }:
+
+stdenv.mkDerivation rec {
+  pname = "pixelorama";
+  version = "0.10.1";
+
+  src = fetchurl {
+    url = "https://github.com/Orama-Interactive/Pixelorama/archive/refs/tags/v${version}.tar.gz";
+    sha256 = "sha256-9Ds62bHkbqsL0Q8VD0SVIEua+PmqmWgPJPxxAGqYVv4=";
+  };
+
+  nativeBuildInputs = [
+    godot-headless
+    godot-export-templates
+  ];
+
+  buildPhase = ''
+    export HOME="$(pwd)"
+    mkdir -p .local/share/godot/
+    ln -s "${godot-export-templates}/share/godot/templates" ".local/share/godot/templates"
+    mkdir -p build
+    godot-headless -v --export "Linux/X11 64-bit" ./build/pixelorama
```

Comment:
Can it also be built on other platforms? Maybe you can make it dynamic with `stdenv.hostPlatform.linuxArch`

If not meta.platform should be “x86_64-linux” only

Reply:
I've adapted it to build on other platforms now, but I have no devices to test this. We will need to wait for the ofborg results to see if it works.

Reply:
Godot itself is currently just built for `i686-linux` & `x86_64-linux` in nixpkgs, thus I've adjusted the platforms.